app/views/post/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="user_comment">
+  <div class="post">
+    <div class="post_comment">
+      <h2><%= @post.title %> by <%= @post.author.name %></h2>
+      <span>Comments: <%= @post.comments_counter %>, Likes: <%= @post.likes_counter %></span>

```

Comment:
[OPTIONAL] 
Please note that the number of comments has doubled.
![image](https://github.com/Mike47ip/Blog-app/assets/83596135/81904bc1-a374-45d7-a00e-712812177b9f)

Because it was not mentioned by the previous reviewer, this aspect wouldn't be mandatory for now, but I highly recommend you fix the bug.